server/paymentProviders/stripe/webhook.ts

```diff
@@ -158,7 +158,12 @@ export const mandateUpdated = async (event: Stripe.Event) => {
 export const paymentIntentSucceeded = async (event: Stripe.Event) => {
   const stripeAccount = event.account ?? config.stripe.accountId;
   const paymentIntent = event.data.object as Stripe.PaymentIntent;
-  const charge = (paymentIntent as any).charges.data[0] as Stripe.Charge;
+
+  let charge = ((paymentIntent as any).charges?.data?.[0] as Stripe.Charge) || paymentIntent.latest_charge;
```

Comment:
In `stripe/creditcard.ts`, we read first `paymentIntent.latest_charge` then fallback on the other. Should we make it the same?